integration/fuse/bin/alluxio-fuse

```diff
@@ -114,18 +120,31 @@ fi
 
 case $1 in
   mount)
-    if [[ $# -eq 2 ]]; then
-      mount_fuse $2 /
+    shift
+    while getopts "o:" option; do
+      case "${option}" in
+        o)
+          mount_option=${OPTARG}
+          ;;
+        *)
+          echo -e "${USAGE}" >&2
```

Comment:
should use USAGE_MSG?

Reply:
it should be `USASGE_MSG` defined https://github.com/Alluxio/alluxio/pull/9497/files#diff-353dcb496e9c07a6d894ece8ecb85c8dR107 @calvinjia

Reply:
ack